helm-chart/sefaria-project/templates/configmap/local-settings.yaml

```diff
@@ -24,3 +24,7 @@ data:
   SERVER_EMAIL: "{{ .Values.localSettings.SERVER_EMAIL }}"
   MULTISERVER_ENABLED: "{{ .Values.localSettings.MULTISERVER_ENABLED }}"
   CRM_TYPE: "{{ .Values.localSettings.CRM_TYPE }}"
+  SENTRY_ENVIRONMENT: {{ .Values.deployEnv | quote }}
+  SENTRY_CODE_VERSION: {{ .Values.web.containerImage.tag }}
+  SENTRY_DSN: {{ .Values.localSettings.SENTRY_DSN | quote }}
```

Comment:
Are these values meant to be secrets? Because if so, I don't think they are defined correctly here.